libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/EventQueues.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    public class EventQueues
```

Comment:
>EventQueues [](start = 17, length = 11)

Can you describe this class?  Is it consumable publicly or should this be internal?

Reply:
I believe this should be public in that it is stored in memory and rules could manipulate it.

---
In reply to: [360169664](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360169664) [](ancestors = 360169664)